templates/layout/_sidebar.html.twig

```diff
@@ -130,4 +130,7 @@
                 '%link_end%': '</a>'
             })|raw }}</p>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+{% include 'layout/_topbar.html.twig' %}
```

Comment:
Top bar as well?

Reply:
This should be fine as it keeps the layout consistent. I'll mark this this one as approved and we can make more changes down the track if needed

Reply:
It doesn't seem to do anything. Even if you enable topbar, it will show at the top (hence the name), but it doesn't got inserted in the footer. I will remove this line I think.